core/platform/models.py

```diff
@@ -300,6 +300,20 @@ def import_search_services(cls):
         from core.platform.search import elastic_search_services
         return elastic_search_services
 
+    @classmethod
+    def import_cloud_translate_services(cls):
+        """Imports and returns cloud_translate_services module.
+
+        Returns:
+            module. The cloud_translate_services module.
+        """
+        if constants.EMULATOR_MODE:
```

Comment:
Remember my comment above: cloud_translate_emulator and cloud_translate_services should expose the same API. Currently, they do not. The first exposes a TranslateEmulator class and the second exposes a translate_text() function.

Please make these consistent. Probably add a translate_text() top-level function to the first one (I'm assuming the functions in the rest of the codebase are going to be calling only that and not invoking the TranslateEmulator directly.

Reply:
I changed this to follow the same pattern as cloud_taskqueue_services. It now uses dev_mode_cloud_translate_services.